### PR TITLE
Stl 2084 informieren page redesign/rename

### PR DESCRIPTION
**Pull request by hamo225**
# Short Description
- This is a redesign of informieren page. We are splitting the information into 2 types. "How it works" and "Help Topics". On the former, we provide the new tutorial video, with clear "how to" steps with images and on the latter a list of all possible questions and answers. 

# Changes
- updates route for /sofunktionierts
- changes menu point in the side navigation to "So funktionierts" 
- adds images and icons
- adds button for the player condition to ButtonAnchor component
- updates general styling 
- creates the HowItWorksComponent + tests + adds to storybook
- creates the HowItWorksPage + tests + adds to storybook
- deletes the info for relatives page everywhere
- creates the helpAreaPage + tests + storybook
- adds plausible tracking goals and methods to buttons
- adds new cta to landing page and eligibility/success page

# Feedback
- Have i added the plausible tracking correctly for the accordion component? and have I also added the plausible methods correctly?
- anything else?

# How to review this Pull Request
[Link to Confluence](https://digitalservicebund.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: webapp/app/templates/components.html [hunk 1]
```diff
@@ -279,3 +279,11 @@ <h5 class="modal-title mb-n2">{{title}}</h5>
         </a>
     </div>
 {% endmacro %}
+
+{% macro eligibilitySuccessBox(heading, textButtonOne, textButtonTwo, urlButtonOne, urlButtonTwo, targetButtonOne, targetButtonTwo , sourceButtonOne, sourceButtonTwo, plausible_domain_set=False, float_right=False) -%}
+    <h2>{{heading}}</h2>
+    <div class="eligibilitySuccessButtons">
+        <a class="{% if  float_right %} float-right {% endif %} btn btn-primary" href="{{ urlButtonOne }}" onclick="plausible('{{ targetButtonOne }}', {props: {method: '{{ sourceButtonOne }}'}})">{{ textButtonOne }}</a>
```
Comment (SannyNguyenHung) [inline webapp/app/templates/components.html:286]: do you need float_right? can't find a reference

File: webapp/app/translations/de/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -5217,27 +5209,57 @@ msgid "maybe_icon"
...
+#: app/templates/eligibility/display_success.html:57
+msgid "tracking.eligibility.how_it_works.success.goal"
+msgstr "So funktionierts"
+
+#: app/templates/eligibility/display_success.html:58
+msgid "tracking.eligibility.help_page.success.goal"
+msgstr "Zum Hilfebereich"
+
+#: app/templates/eligibility/display_success.html:59
+#, fuzzy
```
Comment (SannyNguyenHung) [inline webapp/app/translations/de/LC_MESSAGES/messages.po:5254]: please remove the fuzzy lines

File: webapp/client/src/components/AccordionComponent.js [hunk 3]
```diff
@@ -90,6 +107,9 @@ export default function AccordionComponent({ title, intro, items, variant }) {
                 onClick={() => {
                   if (toggle === index) setToggle(-1);
                   else setToggle(index);
+                  addPlausibleGoal(plausibleDomain, `${item.title}`, {
+                    method: `Hilfebereich /${item.title}`,
```
Comment (SannyNguyenHung) [inline webapp/client/src/components/AccordionComponent.js:111]: `Hilfebereich /${item.title}` the space after Hilfebreich should be removed or?

File: webapp/client/src/lib/faqAnchors.js [hunk 4]
```diff
@@ -61,11 +65,13 @@ const faqAnchorList = [
       <div>
         <p>{t("LandingPage.Accordion.Item2.detail")}</p>
         <ul>
-          <li>{trans("LandingPage.Accordion.Item2.listItem1")}</li>
-          <li>{trans("LandingPage.Accordion.Item2.listItem2")}</li>
+          <li>{t("LandingPage.Accordion.Item2.listItem1")}</li>
```
Comment (SannyNguyenHung) [inline webapp/client/src/lib/faqAnchors.js:68]: are you sure about that? the trans() is used to replace Urls in the localisation

**Review by SannyNguyenHung — CHANGES_REQUESTED** [review]
LGTM! Just some questions

File: webapp/app/templates/components.html (hunk 1 above)
Comment (hamo225) [reply webapp/app/templates/components.html:286]: good point.

File: webapp/client/src/lib/faqAnchors.js (hunk 4 above)
Comment (hamo225) [reply webapp/client/src/lib/faqAnchors.js:68]: yes, design changed the contents of this drop down. No inner urls except in the last p tag. So no need for the trans.

File: webapp/client/src/components/AccordionComponent.js (hunk 3 above)
Comment (hamo225) [reply webapp/client/src/components/AccordionComponent.js:111]: yes it should. but there should also be a space after the slash. Thanks for that. I will adjust this and check the other methods.

**Review by SannyNguyenHung — APPROVED** [review]

**Pull request merged**